README.md

```diff
@@ -1,21 +1,44 @@
-# HOPR
+<p align="center"><a href="https://hoprnet.org" target="_blank" rel="noopener noreferrer"><img width="100" src="https://github.com/hoprnet/hopr-assets/blob/master/v1/logo/hopr_logo_padded.png?raw=true" alt="HOPR Logo"></a></p>
+<h2 align="center">HOPR Chatbot</h2>
 
-HOPR is a privacy-preserving messaging **protocol** which enables the creation of a secure communication network via relay nodes powered by economic incentives using digital tokens.
+**HOPR** is a privacy-preserving messaging **protocol** which enables the creation of a secure communication network via relay nodes powered by economic incentives using digital tokens. **HOPR Chatbot** is a proof-of-concept aimed to showcase the capabilities of a **HOPR Node** by using our gRPC-enabled **HOPR Server**, which listens to messages and replies to the recipients of them, whenever their address is included in the original message.
 
-## chat-bot
+## Overview
+**HOPR Chatbot** is a `node.js` application that connects to an instance of a `hopr-server`, and listens to any calls done against such server. Upon receiving a message with a valid recipient, it provides an answer given the bot that is currently loaded. At the time of writing, **HOPR Chatbot** dynamically loads bots based in the `ENV` given, and each bot has different capabilities.
```

Comment:
As a first time reader stumbling upon this repo, it would be nice to have a link to `hopr-server`

Reply:
A bummer, thought I had it. Good catch, will add this.